spec/commands/startup_spec.rb

```diff
@@ -0,0 +1,38 @@
+require "spec_helper"
+require_relative "../../lib/commands/startup"
+
+describe Commands::Startup do
+  let(:config_directory) { "spec/fixtures" }
+
+  subject { described_class.new(config_directory: config_directory, service: "example-service") }
+
+  let(:run_double) { instance_double(Commands::Run) }
+  before { allow(run_double).to receive(:call) }
```

Comment:
What about adding `allow(subject).to receive(:puts)` here to keep the spec runner output clean?

Reply:
Good idea, I didn't even notice the printing! I've added this now.